dbtmetabase/__init__.py

```diff
@@ -6,112 +6,165 @@
 from .metabase import MetabaseClient
 from .parsers.dbt_folder import DbtFolderReader
 from .parsers.dbt_manifest import DbtManifestReader
+from .models.config import MetabaseConfig, dbtConfig
 
-from typing import Iterable, List, Union
+from typing import Iterable, List, Union, Optional
 
 from ._version import version as __version__
```

Comment:
I get an error from this line. May be related to my setup -- see [other comment](https://github.com/gouline/dbt-metabase/pull/38#issuecomment-885706487) here.

If I comment this line out I can run `dbt-metabase models` with no errors.

Reply:
@gouline 

I also get an error on this line. Wondering if there is a step we are missing in the makefile / setup? 
I did a `pip install -e .` in the local dir which of course finds a setup py. I see in the toml we should be generating it. Just not sure why mine hasnt generated when installing either via `git clone`, `cd`, `pip install -e .` or with `pip install git+https://github.com/z3z1ma/dbt-metabase.git@exposures`

Maybe just something I am missing? Ill test it again in another fresh venv.

Reply:
I will need to document this, but `_version.py` is generated dynamically to allow automatic releases from GitHub Actions. It's not really intended to be installed from Git, you have to run `make` (or `make build`) and if you want to install locally, there's `make dev-install` for that.

However, if you have that use case, I'll think about either including a dummy `_version.py` or making that import optional. I'll experiment and see what works 🤔

Reply:
The version issue should now be fixed in master (see #42). You should be able to install using pip from a git branch, like @z3z1ma wrote above.